packages/blade/tokens/opacities.ts

```diff
@@ -0,0 +1,15 @@
+type opacity = Readonly<{
+  1: number;
```

Comment:
Why 1,2,3,9? Scale seems very weird to me looking at it without context.

Reply:
The scale will be filled up. @saurav12 is working on it.